app/graphql/index.js

```diff
@@ -1,30 +1,64 @@
 const { makeExecutableSchema } = require('graphql-tools'),
   { applyMiddleware } = require('graphql-middleware'),
-  types = require('./types'),
-  inputs = require('./inputs'),
-  users = require('./users'),
-  healthCheck = require('./healthCheck');
+  fs = require('fs'),
+  path = require('path');
 
-const typeDefs = [types, inputs, ...users.schemas, ...healthCheck.schemas];
+const importEverything = () => {
+  const imports = { modules: [] };
+  fs.readdirSync(__dirname, { withFileTypes: true }).forEach(dirent => {
+    if (dirent.isDirectory()) {
+      imports.modules.push({ [dirent.name]: require(path.join(__dirname, dirent.name)) });
+    } else if (dirent.name === 'index.js') {
```

Comment:
at this point, do we need the index.js files or we can delete them?

Reply:
Jmm I dont know if I got you... that statement is for avoiding importing graphql index.js, but it is not necessary. Do you want me to remove it?

Reply:
yes! we should delete them